tests/testlogin.js

```diff
@@ -0,0 +1,80 @@
+//const { assert } = require("console");
+import LoginPage from '../pages/login.page';
+import HomePage from '../pages/home.page';
+
+describe ('TestLogin', async () => {
+
+    it('Should login successfully', async () => {
+
+        await LoginPage.open('login');
+        await LoginPage.loginAs('jsmith@demo.io', 'Demo123!');
+        await expect(HomePage.welcomeUser).toHaveTextContaining('Welcome Josh');
+        
+    });
+
+    it('Should login successfully with Remember me chechbox selected', async () => {
+
+        await LoginPage.open('login');
+        await LoginPage.selectRememberMe();
+        await LoginPage.loginAs('jsmith@demo.io', 'Demo123!');
+        await expect(HomePage.welcomeUser).toHaveTextContaining('Welcome Josh');
+        
+    });
+
+    it ('Should not login with invalid username and correct password', async () => {
+
+        await LoginPage.open('login');
+        await LoginPage.loginAs('andrea@demo.io', 'Demo123!');
+        expect(await LoginPage.statusMessage).toHaveTextContaining('Error');
```

Comment:
In order for this to work, `await` needs to be paced at the begining of the line, instead of before LoginPage. If not, it gives false positives as we discovered during first workshop. Place it there instead.

```javascript
await expect(LoginPage.statusMessage).toHaveTextContaining('Error');

```